source/hovers.js

```diff
@@ -0,0 +1,869 @@
+// import { stringify } from 'querystring';
```

Comment:
Cleanup this dead code.
Dead code (commented code) should never be left in sources.